src/main/java/com/spingboot/demo/repository/WordRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.spingboot.demo.repository;
+
+import com.spingboot.demo.domain.Word;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface WordRepository extends JpaRepository<Word, Integer> {
+    
+    List<Word> findAllByOrderByAmountDesc(Pageable pageable);
```

Comment:
Okay, I see that you have used naming👍Make a few approaches to replace other methods.